core/sql/src/main/java/org/trafodion/sql/HBulkLoadClient.java

```diff
@@ -475,10 +475,12 @@ public boolean release( ) throws IOException {
        writer.close();
        writer = null;
     }
-    if (fileSys !=null)
+    // sss This is one place that is unconditionally closing the 
```

Comment:
Should we remove "sss". It is a minor nit, and I know it is an oversight. Maybe this can be done in a subsequent checkin?